packages/contracts-bedrock/scripts/interfaces/IL2OutputOracle.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { Types } from "../../contracts/libraries/Types.sol";
+
```

Comment:
This will end up out of date - any reason to just not import the actual `L2OutputOracle` instead?

Reply:
Yea agreed, easier not to maintain.